src/sentry/static/sentry/app/stores/tagStore.tsx

```diff
@@ -153,7 +153,8 @@ const tagStoreConfig: Reflux.StoreDefinition & TagStoreInterface = {
   },
 
   onLoadTagsSuccess(data) {
-    Object.assign(
+    this.state = Object.assign(
```

Comment:
This was causing in-place mutation of the tag list which resulted in the searchbar not having the most current data. This issue has been around a _long_ time, but it was really apparent when I was working on the overview.